auth-gateway/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.env
```

Comment:
Perhaps I need to see the Dockerfile and any associated Makefiles, but I'm confused by this.

Why ignore the `.env` file (which, AFAICT, is the filename searched for variable definitions), but still check in the configuration into a file called `public-env`. Now there's an extra step of copying a file from one place to another before this service works?

Reply:
I explain this in the initial comment. .env is used to organize environment variables. Some of those could be secret down the road.

Reply:
The Dockerfile and Makefile will be part of a separate PR. I can add them here, but the goal was to have a single-concern PR. Here the concern is the business logic of the /verify endpoint.

Reply:
OK, we'll revisit this with the makefile and docker file PR.